data/svgs/svg-lookup.ini

```
@@ -170,6 +170,10 @@ Svg=logitech-mx-ergo.svg
 DeviceMatch=bluetooth:046d:b023;usb:046d:4082
 Svg=logitech-mx-master-3.svg
 
+[Logitech MX Master 3S]
+DeviceMatch=bluetooth:046d:b034;usb:046d:c548
```

```suggestion
DeviceMatch=bluetooth:046d:b034
```
Please, remove the receiver ID `c548` (following https://github.com/libratbag/libratbag/pull/1346#discussion_r985252293).